Project/app/modules/controllers/myPlayListController.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+/*****************************************************************************************
+ *  MY PLAY LIST CONTROLLER:                                                             *
+ *  Check user playlist, local stored playlist, create new playlist                      *
+ *****************************************************************************************/
+
+var controller = angular.module('myPlayListCtrl', []);
+controller.controller('myPlayListController',['$scope', 'sessionService', 'List',
+    function($scope, sessionService, List) {
+
+        /* INIT */
+        $scope.myPlayList = sessionService.myPlayList;
+        $scope.index = 0;
+        $scope.list =  $scope.myPlayList.tracks;
+
+        /* REFRESH SCOPES */
+        $scope.$on('$locationChangeSuccess', function (event) {
+            sessionService.ping();
+            $scope.myPlayList = sessionService.myPlayList;
+        });
+        
+        $scope.playLists = {};
+        sessionService.getPlayLists($scope.playLists); //GET THE ONLINE PLAYLIST IT's AUTOMATIC
+        setTimeout(() => $scope.index = 0,1000);
+        
+        /* CHANGES MS TO MINUTES */
+        $scope.msToMinutes = function (time){
```

Comment:
All functions that are kind of generic I'd place them into a library source file